Library/Homebrew/os/mac/sdk.rb

```diff
@@ -0,0 +1,57 @@
+require "os/mac/version"
+
+module OS
+  module Mac
+    class SDK
+      attr_reader :version, :path
+
+      def initialize version, path
+        @version = OS::Mac::Version.new version
```

Comment:
Thanks for the clarification! Being relatively new to the project and not knowing most of the history, I tend to get my information from the code and this is what I (falsely) inferred from it.